src/openloco/GameCommands.cpp

```diff
@@ -227,6 +227,8 @@ namespace openloco::game_commands
                 case element_type::track: // 4
                 {
                     track_element* trackElement = tile->as_track();
+                    assert(trackElement != nullptr);
```

Comment:
I like this idea! If it helps to silence the warning, perhaps you could do the same thing for the `pObject` check below.

You will need to include `<cassert>` though.

Reply:
The objects could 'technically' be null, in case they're nullpointers or something. We know for a fact that the tile won't be null, since we already checked its type.